### DISCUSSION TITLE
square payment gate way squareup.com

### DISCUSSION THREAD

**Discussion by Genovit** [geodesicsolutions-community/geocore-community#164] [General]
will there be a square payment gateway avalable some day for geo classifieds script? for squareup.com payment prossesinng

**Comment by vicos59** [comment]
If I understand it correctly, the intent of the guy who maintains this was to 1) get it released into the public domain; 2) get the licensing stripped out so people could use it without restrictions; and 3) get it updated to a point where it was stable and could be used widely.

1 & 2 are complete and 3 is very close.

The maintainer said that he already has a full life and was not looking to be developing new features. It was his hope that either more coders would come onboard to help develop the app, or site operators would join together to fund some coder to developer new functionality and then offer it back to the repo for everyone to use.

**Reply by Genovit** [reply]
Wow that would be great! if some one created a Square payment gateway for sure!
